install_config/install/advanced_install.adoc

```diff
@@ -402,6 +402,100 @@ meaning that it is available for placement of new pods. See
 xref:marking-masters-as-unschedulable-nodes[Configuring Schedulability on Masters].
 |===
 
+
+
+[[configuring-cluster-pre-install-checks]]
+=== Configuring Cluster Pre-install Checks
+
+Pre-install checks are a set of diagnostic tasks that run as part of the
+*openshift_health_checker* Ansible role. They run prior to an Ansible
+installation of {product-title}, ensure that required inventory values are set,
+and identify potential issues on a host that can prevent or interfere with a
+successful installation.
+
+Checks may have one or more tags, which allow for groups of related checks to be
+run together.
+
+The following table describes available pre-install checks that will run before
+every Ansible installation of {product-title}:
+
+[[configuring-cluster-pre-install-checks-pre-install-checks]]
+.Pre-install Checks
+[options="header"]
+|===
+
+|Check Name |Purpose
+
+|`memory_availability`
+|This check ensures that a host has the recommended amount of memory for the
+specific deployment of {product-title}. Values are taken from the
+xref:../../install_config/install/prerequisites.html#system-requirements[latest
+installation documentation]. A user-defined value for minimum memory
+requirements may be set by passing the variable
+`openshift_check_min_host_memory_gb` to the `openshift_health_checker` role.
+
+|`disk_availability`
+|This check only runs on etcd, master, and node hosts. It ensures that the mount
+path for an {product-title} installation has sufficient disk space remaining.
+Recommended disk values are taken from the
+xref:../../install_config/install/prerequisites.html#system-requirements[latest
+installation documentation]. A user-defined value for minimum disk space
+requirements may be set by passing the variable
+`openshift_check_min_host_disk_gb` to the `openshift_health_checker` role.
+
+|`docker_storage`
+|Only runs on hosts that depend on *docker* (nodes and containerized
+installations). Checks that *docker*'s total usage does not exceed a
+user-defined limit. If no user-defined limit is set, *docker*'s maximum usage
+threshold defaults to `90%` of the total size available. The threshold limit for
+total percent usage can be set with a variable in your inventory file:
+`max_thinpool_data_usage_percent: 90`. A user-defined limit for maximum thinpool
+usage may be set by passing the variable `max_thinpool_data_usage_percent=80%`
+to the `openshift_health_checker` role.
```

Comment:
Maybe s/docker/the docker daemon/ instead? And I believe that the value of `90` is expected (without the `%`)...